### PR TITLE
Student assignment for Python Prep

### PR DESCRIPTION
**Pull request by joelgdwave**
Please consider this code as a student assignment for Python Prep.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+The program ``pythonprep_assignment.py`` is an exercise for students.
+It is a Python exercise designed to test some basic Python skills.
+Our training course includes Python programming and the student needs to be
+able to program in it.
+
+Students are given the file, and need to read the file into a string, and 
+then break the string into numbers, and then sum the numbers.
+
+To run your program type python pythonprep_assignment.py. You have successfully
```
Comment (m3ller) [inline README.rst]: inline code. ` ``python pythonprep_assignment.py`` `

File: pythonprep_assignment.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    ## ------- read file into a string --------
+
+    data = read_file_into_string("Numbers_Assignment_PP.txt")
+
+    ## ------ parse string into list -------
+    number_list = break_file_data_into_list_of_numbers(data)
+
+    ## ------ sum the list -------
+    # TODO: Add code to sum the list
```
Comment (m3ller) [inline pythonprep_assignment.py]: Perhaps also remind students to save their sum to the variable, `sum_of_numbers`?

**Review by m3ller — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (joelgdwave) [reply README.rst]: Yes, thank you.

File: pythonprep_assignment.py (hunk 2 above)
Comment (joelgdwave) [reply pythonprep_assignment.py]: Thanks.

**Pull request merged**